help/downloads.md

```diff
@@ -65,6 +65,8 @@ Here are the main sections of our FTP site, with links to README files and help
 | Previous releases | [ftp](https://ftp.uniprot.org/pub/databases/uniprot/previous_releases/) |
 
 # Embeddings
+[Parent directory](https://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/embeddings/)  
+[README](https://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/embeddings/README)  
```

Comment:
Wondering whether we should add a row for embeddings resource in the first table for UniProtKB, similar to others?

Reply:
this is done in the next commit